[MS] Update no organization homepage (#10718)
Scille/parsec-cloud: docs/userguide/security.rst

When you receive a link, you just need to check if the ``SERVER`` and ``ORGANIZATION`` parts match your organization's server. This can be found on the "Information" page, when you login to your organization.

When joining an organization, you should contact an administrator of the organization to provide or confirm this information.

How to enable browser storage for Parsec?
=========================================

> To enable browser storage for Parsec, you need to allow the browser to store data locally.

Reviewer: I'd add a little introduction.

```suggestion
Parsec needs to securely store your access keys (device) on your web browser so you can login to your organizations.

This section explains how to enable storage for Parsec depending on your web browser.
```

Chrome and Chromium-based browsers
----------------------------------

1. Go to the settings menu (three dots in the upper right corner).
2. Click on "Privacy and security."
3. Click on "Site settings."
4. Under "Content", click on "Cookies and site data."

> 5. Make sure "Allow sites to save and read cookie data" is enabled.

Author: I do have it on Arc browser (Chromium based-browser) :

> Based on Chromium version 138.0.7204.158 (Official Build) (arm64)

<img width="1077" height="792" alt="image" src="https://github.com/user-attachments/assets/af8e45cd-33d1-4663-ba5a-1dc0291a9055" />

> 5. Make sure "Allow sites to save and read cookie data" is enabled.

Author: But if you don't have it, I can put only Chrome. I don't think the user will be lost, this is probably with the same wording, neither do I?

Firefox
-------

1. Go to the settings menu (horizontal lines in the upper right corner).
2. Click on "Privacy & Security."
3. Under "Content," click on "Cookies and site data."

> 4. Make sure "Allow sites to save and read cookie data" is enabled.

Author: Version `140.0.4`. 

"Cookies and Site Data" -> This is a section, I updated it.

When you receive a link, you just need to check if the ``SERVER`` and ``ORGANIZATION`` parts match your organization's server. This can be found on the "Information" page, when you login to your organization.

When joining an organization, you should contact an administrator of the organization to provide or confirm this information.

> How to enable web browser storage?
> ==================================

Reviewer: Remove question marks (to be consistent with the other sections)
```suggestion
How to enable web browser storage
=================================
```

Parsec needs to securely store your access keys (device) on your web browser so you can login to your organizations.

This section explains how to enable storage for Parsec depending on your web browser.

Chrome

> ----------------------------------

Reviewer: ```suggestion
------
```
